public/js/safetymaps/modules/vrgz/route.js

```diff
@@ -0,0 +1,131 @@
+/*
+ *  Copyright (c) 2020 B3Partners (info@b3partners.nl)
+ *
+ *  This file is part of safetymapDBK
+ *
+ *  safetymapDBK is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  safetymapDBK is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with safetymapDBK. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+var dbkjs = dbkjs || {};
+window.dbkjs = dbkjs;
+dbkjs.modules = dbkjs.modules || {};
+
+dbkjs.modules.route = {
+    id: "dbkjs.modules.route",
+    options: null,
+    reloadTimeOut: null,
+    routeUrlSimacar: "/simacar_export/Route.txt",
```

Comment:
Do you have a test file with a route so I can test it? Maybe put it in a `test` directory next to `public` at the root of the repository.